scrapy/settings/default_settings.py

```diff
@@ -264,3 +264,5 @@
     'scrapy.contracts.default.ReturnsContract': 2,
     'scrapy.contracts.default.ScrapesContract': 3,
 }
+
+HTTPPROXY_ENCODING = 'latin-1'
```

Comment:
What do you think about making setting name more explicit, e.g. HTTPPROXY_AUTH_ENCODING?

Settings are sorted alphabetically, so this option should be moved up.